news-analysis.py

```diff
@@ -130,29 +130,9 @@ def ticker_socket(msg):
     conn_key = bsm.start_symbol_ticker_socket(coin+PAIRING, ticker_socket)
 bsm.start()
 
-
-def find_lot_size():
```

Comment:
Why has this been removed and how do you account for different step sized in order? ie: BTC is 0.000001 and XRP is 0.1. The incorrect format volume will raise an exception and fail to place the order.

Reply:
This was fixed in commit 394598e by rybens92.